src/sql/workbench/services/accountManagement/browser/accountManagementService.ts

```diff
@@ -279,10 +279,14 @@ export class AccountManagementService implements IAccountManagementService {
 
 		let finalResult = true;
 		for (const account of accounts) {
-			const removeResult = await this.removeAccount(account.key);
-			if (removeResult === false) {
-				this._logService.info('Error when removing %s.', account.key);
-				finalResult = false;
+			try {
+				const removeResult = await this.removeAccount(account.key);
+				if (removeResult === false) {
+					this._logService.info('Error when removing %s.', account.key);
```

Comment:
Huh, I guess I wasn't even aware that this syntax was supported. You've tried it out for all the various logger outputs? Might be best just to use interpolation just to keep consistent with the rest of the app regardless...